charts/cofide-connect/values.yaml

```diff
@@ -31,6 +31,9 @@ connect:
     OTEL_METRIC_EXPORT_INTERVAL: 60000
     OTEL_SERVICE_NAME: connect
 
+dashboard:
```

Comment:
nit: these are the allowed origins for connect, of which the dashboard is typically one. So it's probably better under `connect`.

Reply:
Good point, moved